Cargo.toml

```diff
@@ -337,7 +337,6 @@ thiserror = "1.0.59"
 time = { version = "0.3.36" }
 unicode-segmentation = "1.11.0"
 unicode-width = "0.1.12"
-utf-8 = "0.7.6"
```

Comment:
I don't yet see why this PR needs to touch windows-sys and utf-8. There's probably a good reason that I just don't see. Could you fill me in?

Reply:
I probably removed it after running udeps. Looking back at it now, we need both of them for windows specific function and i definitely ignored it since the CI passed.

Reply:
@BenWiederhake  I was wondering, would basically updating my master and then merging to my feature branch be okay to do ? I kinda need some help on how to rebase this, can't seem to figure it out.